install.rst

```diff
@@ -336,6 +336,19 @@ If you want to have a visual overview of your API in your browser you can add sw
 
 With this you can see the swagger-ui in your browser on port 8080.
 
+Deploying to Heroku
+===================
+Assuming your making modifications locally and then pushing to GitHub, it's easy to deploy to Heroku.
+ 
+1. Create a new app on Heroku
+2. In Settings add the following buildpack :code:`https://github.com/PostgREST/postgrest-heroku`
+3. Add the require Config Vars in Heroku (see above)
```

Comment:
Instead of (see above) maybe we could add the list of vars we have in https://github.com/PostgREST/postgrest/blob/master/app.json#L7-L57

Reply:
Sorry, I meant to add a link to that list, in reStructuredText this is done with:
```
`list of config vars <https://github.com/PostgREST/postgrest/blob/master/app.json#L7-L57>`_
```